src/mixins/legal-api-mixin.ts

```diff
@@ -131,4 +131,52 @@ export default class LegalApiMixin extends Mixins(FilingTemplateMixin) {
     const config = { baseURL: authUrl }
     return axios.get('users/@me', config)
   }
+
+  /**
+   * Fetch business data for current business.
+   * @param datatype The type of data to request.
+   * @returns a promise to return the business base info or of the specified data type.
```

Comment:
Small typo.

Reply:
I see no spelling mist.... 😆  Fixed in next commit!

Reply:
Grate!